contracts/staking/slashing/SlashingManager.sol

```diff
@@ -29,47 +29,37 @@ contract SlashingManager is ISlashingManager, Ownable {
         heimdallId = keccak256(abi.encodePacked(_heimdallId));
     }
 
-    function updateSlashedAmounts(
-        address proposer,
-        bytes memory vote,
-        bytes memory sigs,
-        bytes memory slashingInfoList,
-        bytes memory txData
-    ) public {
-        RLPReader.RLPItem[] memory dataList = vote.toRlpItem().toList();
-        require(
-            keccak256(dataList[0].toBytes()) == heimdallId,
-            "Chain ID is invalid"
+    function updateSlashedAmounts(bytes memory data, bytes memory sigs) public {
+        (
+            uint256 _slashingNonce,
+            address proposer,
+            bytes memory _slashingInfoList
+        ) = abi.decode(
+            data,
+            (
+                uint256,
+                address,
+                bytes
+            )
         );
-        require(dataList[1].toUint() == VOTE_TYPE, "Vote type is invalid");
-        require(slashingNonce < dataList[2].toUint(), "Invalid slashing nonce");
-        slashingNonce = dataList[2].toUint();
+
+        // require(slashingNonce < dataList[2].toUint(), "Invalid slashing nonce");
+        // slashingNonce = dataList[2].toUint();
```

Comment:
Remove this comments if we don't need.

Reply:
maybe @0xAshish  needs it, let's wait for him.

Reply:
maybe @0xAshish needs it, let's wait for him.